src/driver-hidpp20.c

```diff
@@ -343,6 +343,41 @@ hidpp20drv_write_button(struct ratbag_button *button,
 	return -ENOTSUP;
 }
 
+static int hidpp20drv_write_led(struct ratbag_led *led,
+    enum ratbag_led_mode mode,
+    struct ratbag_color color, unsigned int rate,
+    unsigned int bright) {
+	struct ratbag_profile *profile = led->profile;
+	struct ratbag_device *device = profile->device;
+	struct hidpp20drv_data *drv_data = ratbag_get_drv_data(device);
+	struct hidpp20_profile *h_profile;
+	struct hidpp20_led *h_led;
+
+	/* if (!(drv_data->capabilities & HIDPP_CAP_BUTTON_KEY_1b04)) */
+	/* 	return -ENTSUP; */
```

Comment:
logitech drivers sends between 16000 - 20000 values, 16000 being 1 sec cycle and 20000 being 20 sec cycle,
i have no idea what kind of unit it is.